utils/hct/hctdb.py

```diff
@@ -1043,9 +1043,70 @@ def populate_categories_and_models_ExperimentalOps(self):
         # Note: Experimental ops must be set to a shader model higher than the
         # most recent release until infrastructure is in place to opt-in to
         # experimental ops and the validator can force use of the PREVIEW hash.
-        for i in "ExperimentalNop".split(","):
-            self.name_idx[i].category = "No-op"
-            self.name_idx[i].shader_model = 6, 10
+        for name in "ExperimentalNop".split(","):
+            i = self.name_idx[name]
+            i.category = "No-op"
+            i.shader_model = 6, 10
+
+        # Group Wave Index / Count
+        for name in "GetGroupWaveIndex,GetGroupWaveCount".split(","):
+            i = self.name_idx[name]
+            i.category = "Wave"
+            i.shader_model = 6, 10
+
+        # Clustered Geometry
+        for name in ("ClusterID").split(","):
```

Comment:
```suggestion
        for name in "ClusterID".split(","):
```
the parens are weird here right?

Reply:
While I'm starting a new pattern, I should make it nicer... Updating...